packages/field-base/src/input-mixin.js

```diff
@@ -219,5 +220,15 @@ export const InputMixin = dedupingMixin(
         // Setting a value programmatically, sync it to input element.
         this._forwardInputValue(newVal);
       }
+
+      /**
+       * Indicates whether the value is different from the default one.
+       * Override if the `value` property has a type other than `string`.
+       *
+       * @protected
+       */
+      get _hasValue() {
```

Comment:
Let's add this also to `input-mixin.d.ts` as it's a protected API.

Reply:
Done.